packages/stripe-graphql-js/src/schema/stripe.ts

```diff
@@ -7,6 +7,35 @@ import { stripe } from '../utils'
 
 builder.objectType('Stripe', {
   fields: (t) => ({
+    connectedAccounts: t.field({
+      type: 'StripeConnectedAccounts',
+      resolve: async (_parent, _, context) => {
+        const { isAdmin } = context
+
+        if (!isAdmin) throw new GraphQLYogaError('Not allowed')
```

Comment:
Yes, I had a hard time deciding on this.
For an application who was individual users, I would consider connected accounts not be to a source that would be connected to the users. It's rather source for the platform, and then some specific handpicked data could be exposed/mirrored to the users.

But it could just as well be that you want the users to have total transparency to the connected accounts without having to expose or mirror any specific data.

One solution could be to have a separate `isAllowed` who only returns the connected account if you're an admin or if the user has any connection to the connected account(e.g could be found via email).

Since Hasura still has very buggy permission handling for remote schemas, and the manual tinkering with metadata to get around it is something a lot of users might not be confident with, setting it to admin only was the conclusion I came too.

But I don't have a very strong opinion on it if you believe that keeping the permissions consistent between all the root fields is the way to go.

Reply:
Scratch my previous comment. 
Hasura is actively working on fixing the permissions for the upcoming release. I'll swap the isAdmin out for the isAllowed route later today

Reply:
Do you think the current `isAllowed` function works here, given it's relying on a stripe customer id? 

When listing customers, we use the `isAllowed` function to filter the customers:

https://github.com/nhost/nhost/blob/35a6dc6d8999c6ed7ae7e12ba06efa5cc8215cdd/packages/stripe-graphql-js/src/schema/stripe.ts#L67-L74

I'm not sure if we can do something similar here?

Reply:
I ended up thinking the same thing.
The responsibility to request this source is better to move out of the package.Hasuras permission handling offers finegrained handling enough IMO

Reply:
@elitan Would you mind taking another look at this? :)

Reply:
I think we can get back to using the `isAdmin` for now unless we have any better ideas. And we can come back later and improve if needed.

Reply:
Since Hasura have fixed their remote schema handling I believed that it might be enough.
But lets go the way you suggested. I've added the `isAdmin` checks again and if there's any feedback on usage we can rework it then